index.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Portfolio Mobile First</title>
+  </head>
+
+  <body>
+    <!-- Header-section starts here -->
+ <div class="hero-section">
+
+    <header>
+      <div class="container">
+       
+        <div class="mobile-status">
+          <div class="left-side">
+            <h4>9:14</h4>
+          </div>
+          <div class="right-side">
+              <img src="/images/status-icons.svg" alt="status icons" />
+            </div>
+          </div>
+          <nav class="navbar">
+            <div class="logo">
+              <h3>MOZO</h3>
```

Comment:
- Please, let's make this an anchor tag as we'd want the users to always go back to the home page if they visited the other page in your portfolio.